rackspace/cdn/v1/base/delegate_test.go

```diff
@@ -0,0 +1,46 @@
+package base
+
+import (
+	"testing"
+
+	os "github.com/rackspace/gophercloud/openstack/cdn/v1/base"
+	th "github.com/rackspace/gophercloud/testhelper"
+	fake "github.com/rackspace/gophercloud/testhelper/client"
+)
+
+func TestGetHomeDocument(t *testing.T) {
+	th.SetupHTTP()
+	defer th.TeardownHTTP()
+	os.HandleGetSuccessfully(t)
+
+	actual, err := Get(fake.ServiceClient()).Extract()
+	th.CheckNoErr(t, err)
+
+	expected := os.HomeDocument{
+		"rel/cdn": `{
+        "href-template": "services{?marker,limit}",
+        "href-vars": {
+            "marker": "param/marker",
+            "limit": "param/limit"
+        },
+        "hints": {
+            "allow": [
+                "GET"
+            ],
+            "formats": {
+                "application/json": {}
+            }
+        }
+    }`,
```

Comment:
Same as the OpenStack comment about this test: it's weird to me how this document is being parsed only a single level down.